lib/stack_master/commands/lint.rb

```diff
@@ -0,0 +1,42 @@
+require 'tempfile'
+
+module StackMaster
+  module Commands
+    class Lint
+      include Command
+      include Commander::UI
+
+      def initialize(config, stack_definition, options = {})
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def perform
+        unless cfn_lint_available
+          puts "Failed to run cfn-lint, do you have it installed and available in $PATH?"
+          return
+        end
+
+        Tempfile.open(['stack', '.json']) do |f|
+          f.write(proposed_stack.template_body)
```

Comment:
An alternative to using files could be to send the template as STDIN to cfn-lint (assuming it supports that).